### ISSUE TITLE
Suggestion: Reduce Find a Joomla! Extension to just 1 level

### ISSUE THREAD

**Issue by coolcat-creations** [dneukirchen/joomla-org-template#23]
The dropdown is way too long and confusing

**Comment by dneukirchen** [comment]
I copied the dropdown from: http://extensions.joomla.org/extensions/extension.
Its the jed category hierarchy. I totally agree with you, the dropdown is not very user friendly, but i would not change it in the template. 

what do the others think? @BrittaStender @aDaneInSpain ... ?

**Comment by aDaneInSpain** [comment]
I agree that it is very long. But it does the job. It helps show how many different categories we have as well as allows users to quickly navigate to the place they are interested in. I don't think there is anything to do here at the moment.

**Comment by aDaneInSpain** [comment]
We COULD do what @designbengel is suggesting and only show level 1. I think it would be a shame as this is not really meant to be a tool (you go to the JED directly if you know about it).

It is meant to show for NEW users who are not familiar with Joomla that we have many extensions in many categories.

**Comment by dneukirchen** [comment]
@aDaneInSpain good idea.
